thirdeye/thirdeye-frontend/app/pods/components/rootcause-chart/component.js

```diff
@@ -219,6 +231,31 @@ export default Ember.Component.extend({
     }
   },
 
+  _transformSeries(mode, series) {
+    switch(mode) {
+      case 'absolute':
+        return series; // raw data
+      case 'relative':
+        return this._transformSeriesRelative(series);
+      case 'log':
+        return this._transformSeriesLog(series);
+    }
+    return series;
+  },
+
+  _transformSeriesRelative(series) {
+    const first = series.values.filter(v => v)[0];
```

Comment:
Let's try using better variable names that just letters for readability. The compiler would shorten them anyway for performance reason.

Reply:
what would be a readable suggestion here? :D